src/variety/cbblock.js

```diff
@@ -177,8 +258,27 @@ Encode:{
 		this.outbstr += cm;
 	}
 },
+
+"Encode@dbchoco":{
+	decodePzpr : function(type){
```

Comment:
just curious, do you have an idea why the dedicated method? (I realize it's done for cbblock and agree it should be handled the same for both, just don't see a reason to not just inline)

Reply:
I have no idea 😄 

In the shared files, many actions are spread across several methods, presumably to make it easier to override them in specific puzzles. It doesn't make sense to me to do the same inside specific puzzle files, since they won't be overridden any further. I followed the example of cbblock in this file, but I just used one method in Pencils.